Dockerfile

```diff
@@ -20,8 +20,8 @@ COPY pkg/sdk/go.* /build/pkg/sdk/
 RUN go mod download
 
 ARG VERSION
-ARG COMMIT_HASH
-ARG BUILD_DATE
+ARG COMMIT_HASHH
+ARG BUILD_DATEE
```

Comment:
**Question**: what's the reasoning behind the deliberate misspelling? I guess it clashes with some other names, but I might propsose adding a `DOCKER_ARG_` prefix or something similar, or maybe updating the other reference.

Reply:
I was testing something, but I reverted the change since then.